custom_components/google_home/models.py

```diff
@@ -83,6 +94,15 @@ def get_next_timer(self) -> Optional[GoogleHomeTimer]:
         return timers[0] if timers else None
 
 
+class GoogleHomeTimerDict(TypedDict):
+    """Typed dict representation of Google Home time"""
+
+    timer_id: str
+    fire_time: int
+    duration: str
+    label: Optional[str]
```

Comment:
I suggest we move out `TypedDict` to a separate file like `types.py`. It would give it a bit better structure